### DISCUSSION TITLE
What's the proper way to define a "MAC Address" for a column type?

### DISCUSSION THREAD

**Discussion by waliddoch** [fastapi/sqlmodel#978] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the SQLModel documentation, with the integrated search.
- [X] I already searched in Google "How to X in SQLModel" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to SQLModel but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to SQLModel but to [SQLAlchemy](https://github.com/sqlalchemy/sqlalchemy).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python

from sqlmodel import SQLModel, Field, Relationship
import sqlalchemy as sa 

class SensorDevice(SQLModel, table=True):
    __tablename__: str = "sensor_device"
    
    id: int | None = Field(default=None, primary_key=True)
    mac_address: str = Field(sa_column=sa.Column(sa.types.MACADDR))  # Using MACADDR type

```

### Description

What is the common proper way to set a MAC Address via SQLModel (PostgreSQL) ? 
For the following code example, i got this error: 

> "MACADDR" is not a known attribute of module ".types"PylancereportAttributeAccessIssue
> (function) MACADDR: Unknown

### Operating System

Linux, Windows, macOS

### SQLModel Version

0.0.19

### Python Version

Python 3.12.4

**Comment by YuriiMotov** [comment, marked as answer]
This type is dialect-specific (postgresql).
Use correct import path

```py
from sqlalchemy.dialects.postgresql import MACADDR
```
